nemesis/arena_prep.py

```diff
@@ -961,17 +961,17 @@ def one_mp_ready():
 
 def tonberry_levels_battle():
     screen.await_turn()
-    tidus_turn = False
+    tidus_AP_gained = False
     while memory.main.battle_active():
         if memory.main.turn_ready():
             if screen.turn_tidus():
-                if tidus_turn == True:
+                if tidus_AP_gained == True:
                     battle.main.tidus_flee()
-                elif memory.main.get_overdrive_battle(character=0):
+                elif memory.main.get_overdrive_battle(character=0) == 100:
```

Comment:
Moving forward we should be doing:

```python
from players import Tidus
...
elif Tidus.has_overdrive():
```

Not worth holding this up, but a note for the future.